testdata/structtest/struct_test.c

```diff
@@ -361,3 +361,19 @@ struct FloatAndBool {
 int FloatAndBool(struct FloatAndBool f) {
     return f.has_value;
 }
+
+struct FourInt32s {
+    int32_t f0;
+    int32_t f1;
+    int32_t f2;
+    int32_t f3;
+};
+
+#include <stdio.h>
+#include <stdlib.h>
+
+char* FourInt32s(struct FourInt32s s) {
+    char* result = malloc(64);
```

Comment:
Do we care to free this on the go side?

Reply:
I tend to be OK to not care about memory leaks in the tests, but as we have already done by passing buffers from the Go side in other tests, what about following the same way for consistency?